### ISSUE TITLE
Dynamic content support

### ISSUE THREAD

**Issue by Cally99** [Gerapy/Gerapy#106]
Hi there, 

I just found this repo. I was wondering does this repo support splash for spiders? is there any documentation for this repo to show how you configure splash?

Thanks

**Comment by Germey** [comment]
Yes — Gerapy depends on `scrapy-splash`, and `gerapy-pyppeteer` is available for headless-browser rendering. Configure the Splash middleware / `SPLASH_URL` in your Scrapy project as usual. See https://docs.gerapy.com. Closing as supported.
